dlrover/trainer/torch/elastic_dataloader.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2023 The DLRover Authors. All rights reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+
+from torch.utils.data import BatchSampler, DataLoader
+
+
+class ElasticDataLoader(DataLoader):
```

Comment:
Add a docstring to explain the `ElasticDataLoader` like https://github.com/pytorch/pytorch/blob/main/torch/utils/data/dataloader.py#L123-L211

Reply:
+1